Extensions/DateTimeExtensions.cs

```diff
@@ -10,6 +10,14 @@ public static class DateTimeExtensions
 
         public static string ToIsoDateString(this DateTime dateTime) => dateTime.ToString("yyyy-MM-dd");
 
+        public static string ToIsoDateString(this DateTime? dateTime) =>
+            dateTime.HasValue ? dateTime.Value.ToIsoDateString() : string.Empty;

```

Comment:
There is a more elegant way:
```suggestion
            dateTime?.ToIsoDateString() ?? string.Empty;

```